src/lib/include/ops.h

```diff
@@ -2,7 +2,7 @@
 #define DISTOPIA_OPS_H
 
```

Comment:
Can we kill this header off? Ops doesn't really tell me what it even does?

Reply:
While we can definitely rename, as far as I understand this is the most critical header we have as it implements all the distance calculations as templates for *all* the types we are using, including scalars and packed. 

@nbgl may be able to shed more light on this but as I understand you only need to provide all the correct operations on your type and `Distance3DWithBoundary()` does the rest?